### DISCUSSION TITLE
How to move a button and a lview link to the right of the data provider(table)

### DISCUSSION THREAD

**Discussion by Etkiny** [Budibase/budibase#4999] [Help] [answered]
Hello,

I have a table from MySQL and I added a button and a view link to manipulate the table. The problem is I want to move this button and view link to the right end of the table, however they are on the left of the table as default and I could not move them to right. 

How can I move the button and view link to somewhere I want. Here is a ss to make it clear.

![tempsnip](https://user-images.githubusercontent.com/83818438/159477059-ece6289f-6553-49d4-8ce6-eb0c4b0d9a5b.png)

**Comment by aptkingston** [comment, marked as answer]
Hey @Etkiny. You can't currently do this, but I'll raise it as a feature request!

Ticket: https://github.com/Budibase/budibase/issues/5005

**Reply by Etkiny** [reply]
Thank you :)
